ci/runner/Dockerfile

```diff
@@ -37,7 +37,7 @@ SHELL ["/bin/bash",  "-c"]
 RUN apt update && \
     DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC \
     apt install --no-install-recommends -y \
-        libnuma1 && \
+    libnuma1 && \
```

Comment:
This indentation was intentional since it was a continuation of the same command (`\`) instead of a new command (`&& \`)

Reply:
Neither was my change, VSCode is applying it. I'll look into which extension/setting is enforcing this.